modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/kb_article_suggestions.jsp

```diff
@@ -85,7 +85,7 @@ if (ratingsType == null) {
 		<c:when test="<%= kbCommentsCount == 1 %>">
 			<c:choose>
 				<c:when test="<%= showAdminSuggestionView %>">
-					<h5>
+					<h5 class="mb-0">
```

Comment:
Hey @jaimelr10, just saw you added this class here. It is inside a conditional and I guess in the other condition (few lines below) this should be also neccesary, am I wrong? 
```
<c:when test="<%= kbCommentsCount > 1 %>">
			<c:choose>
				<c:when test="<%= showAdminSuggestionView %>">
					<h5>    <=====
						<liferay-ui:message arguments="<%= kbCommentsCount %>" key="there-are-x-suggestions" />

						<c:if test="<%= pendingKBCommentsCount > 0 %>">
							(<liferay-ui:message arguments="<%= pendingKBCommentsCount %>" key="x-pending" />)
						</c:if>
					</h5>
				</c:when>
				<c:otherwise>
					<liferay-ui:message arguments="<%= kbCommentsCount %>" key="you-sent-x-suggestions-for-this-article" />
				</c:otherwise>
			</c:choose>
		</c:when>
```